main.js

```diff
@@ -2,5 +2,127 @@
  * Created by Jenya on 18.04.15.
  */
 
+/// At first I'm creating general BeingClass. It will be the top of all being classes.
+function BeingClass(clas) {
+    this.clas = clas;
+    this.movementType = move.run;
+    this.legs = 0
+}
+
+// Than I'm adding move function to all beings.
+BeingClass.prototype.makeMove = function () {
+    this.movementType.move(this.name);
```

Comment:
var 1 simply doesn't work.

    BeingClass.prototype.makeMove = function () {
    this.movementType(this.name);
    }

as `movementType` is not a function.